apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -384,4 +392,9 @@ protected static int getNumParallelJobs(CommandLine cmdLine) {
     }
     return ret;
   }
+
+  protected static int getTaskTimeOut(CommandLine cmdLine) {
+    return NumberUtils.toInt(
+        cmdLine.getOptionValue("taskTimeOutInHours"), DEFAULT_TASKS_WAIT_HOURS);
```

Comment:
```suggestion
        cmdLine.getOptionValue("tasksWaitHours"), DEFAULT_TASKS_WAIT_HOURS);
```
Should this be `tasksWaitHours`?